### ISSUE TITLE
[Namespaces.UnusedUses] False negative for multiple imports with single `use` keyword 

### ISSUE THREAD

**Issue by MercierCorentin** [slevomat/coding-standard#952]
I have this example php file:
```php
use Foo\{
        A, B
};

use Bar\{
        C, D
};

class Test {
        public test(){
                return A::returnSomething();
        }

        public test2(){
                return D::returnSomething();
        }
}
```
Composer.json:
```json
{
    "require-dev": {
        "slevomat/coding-standard": "^6.1"
    }
}
```
Command:
```bash
./vendor/squizlabs/php_codesniffer/bin/phpcs --standard=phpcs.xml test.php
```
The result I get:
```
FILE: /home/oem/sandbox/testCS/test.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 7 | ERROR | [x] Type C is not used in this file.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 19ms; Memory: 4MB
```
It think should report that `B` is not used !
Also, when I run this command to remove unused imported class:
```bash
./vendor/squizlabs/php_codesniffer/bin/phpcbf --standard=phpcs.xml test.php
```
It removes a used classe:
```php
<?php

use Foo\{
	A, B
};


class Test {
	public test(){
		return A::returnSomething();
	}

	public test2(){
		return D::returnSomething();
	}
}
```

**Comment by kukulich** [comment]
Yes, we don't support group `use` statements.

**Comment by MercierCorentin** [comment]
Will it be supported in the future?

**Comment by kukulich** [comment]
The group `use` statement syntax is bad and should not be used so I don't want to invest my time to support it. However, if someone prepare PR that will add support for this syntax to all affected sniffs I will merge it.
